components/playlist/resources/components/app.tsx

```diff
@@ -96,7 +104,8 @@ export class PlaylistPage extends React.Component<Props, State> {
                         : this.getImgSrc(item.id)}/>
             )
           },
-          { content: (<span>{item.ready ? 'Ready' : 'Downloading'}</span>) },
+          { content: (<span>{item.cached ? 'Cached' : 'Not cached'}</span>) },
+          { content: (<button style={this.lazyTextButtonStyle} onClick={this.onClickDataButton.bind(this, item.id)}>{item.cached ? 'Remove cache' : 'Cache'}</button>) },
```

Comment:
`onClickDataButton` is an arrow function, so I think this can be
```suggestion
          { content: (<button style={this.lazyTextButtonStyle} onClick={() => this.onClickDataButton(item.id)}>{item.cached ? 'Remove cache' : 'Cache'}</button>) },
```

Reply:
Cool! Done.